src/include/containers/List.hpp

```diff
@@ -34,11 +34,13 @@
 /**
  * @file List.hpp
  *
- * A linked list.
+ * An intrusive linked list.
  */
 
 #pragma once
 
+#include <stdlib.h>
```

Comment:
Out of interest, what's used here from `stdlib.h`?

Reply:
size_t